packages/backend/src/scheduler/SchedulerTask.ts

```diff
@@ -1266,11 +1266,14 @@ export default class SchedulerTask {
             case ThresholdOperator.INCREASED_BY:
             case ThresholdOperator.DECREASED_BY:
                 const secondValue = getValue(1);
-                const increase = firstValue - secondValue;
                 if (operator === ThresholdOperator.INCREASED_BY) {
-                    return thresholdValue < increase / (secondValue * 100);
+                    const percentageIncrease =
+                        ((firstValue - secondValue) / secondValue) * 100;
+                    return percentageIncrease > thresholdValue;
                 }
-                return thresholdValue > increase / (secondValue * 100);
+                const percentageDecrease =
+                    ((secondValue - firstValue) / secondValue) * 100;
+                return percentageDecrease > thresholdValue;
```

Comment:
Nitpick: can we rename `firstValue` and `secondValue` to something like `latestValue` and `previousValue`?

It might be a bit counter intuitive to name them `first` and `second` imo

Feel free to merge as is if this doesn't make sense 🚀